.azure-pipelines-templates/azure_cli.yml

```diff
@@ -2,6 +2,9 @@ steps:
   - script: |
       set -ex
       curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+      az extension add --source https://acccliazext.blob.core.windows.net/confcom/confcom-0.2.5-py3-none-any.whl -y
```

Comment:
After the extension being in public preview, we can install the latest version automatically by `az extension update --name confcom`.
But for now we need to manually manage the version (currently 0.2.5)

Reply:
Could you add this as a comment in the code too?